scripts/run_check_release_notes

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+.+\*\(.+\)\*"
+if [ $? != "0" ]
+then
+	L=git diff origin/master -- doc/news/_preparation_next_release.md | egrep "^\+" | wc -l
+	if [ "$L" -ge "5" ]
+	then
+		echo "We assume that you know what you are doing."
+		exit 0
+	fi
+	echo "The release notes were not extended correctly (with $L lines)."
+	echo "Please make sure you added at least one line which says *(your name)* to"
+	echo "make sure that your contribution is correctly attributed in the release notes."
+	echo ""
```

Comment:
The quotes after `echo` are useless, but I guess that is a matter of style preference. I also would prefer single quotes whenever possible, since using double quotes often has [unintended consequences](https://github.com/ElektraInitiative/libelektra/commit/aace3610003c184af7e6e9ec4d7f3f2f05b1d47f).

Reply:
I think they are quite useful if you start to write new lines to be echoed.